### PR TITLE
Project #3 Positioning and Floating

### PR DESCRIPTION
**Pull request by cvilla714**
On this branch, we created all of the code, the top navigation bar, the content for the page, the side images, bottom navigation bar, and footer as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,30 @@
...
-# Project Name
+# Position and Floating
 
-> One paragraph statement about the project.
+Project #3 Cloning a NY Times article
 
-![screenshot](./app_screenshot.png)
+![screenshot](./app_screenshot.png/)
 
 Additional description about the project and its features.
```
Comment (kevcap) [inline README.md]: Add a description here

File: README.md [hunk 2]
```diff
@@ -1,76 +1,30 @@
...
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://cvilla714.github.io/floating/)
 
+👤 **kc**
 
-## Getting Started
+- Github: [@githubhandle](https://github.com/cvilla714)
+- Twitter: [@twitterhandle](https://twitter.com/kckeyti)
+- Linkedin: [linkedin](https://www.linkedin.com/in/cosmel-villalobos-1900531aa/)
```
Comment (kevcap) [inline README.md]: inside the [  ] add your social medias. e.g.: Instead of @githubhandle add your @username

File: README.md [hunk 3]
```diff
@@ -1,76 +1,30 @@
...
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](lic.url) licensed.
+- Github: [@githubhandle](https://github.com/abmasadullah)
+- Twitter: [@twitterhandle](https://twitter.com/abmasadullah)
+- Linkedin: [linkedin](https://www.linkedin.com/in/abmasadullah/)
```
Comment (kevcap) [inline README.md]: inside the [  ] add your social medias. e.g.: Instead of @githubhandle add your @username

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

  ### Hi team, you guys did a very good job so far!
  #### I can see that you guys are working hard at this project, and you're almost done, there are just some required changes* that you guys have to implement in order to proceed:

  _Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, in order to get more knowledge and have a better project to add at your portfolio.
  
## [REQUIRED CHANGES]
#### Clone REQUIREMENTS 👥 (Those adjusts to make your page really look like the real one)⤵
- [x] The NY Times logo should be a link (link to page) ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png)from Pathwright)
- [x] Please Increase the width of your text column and use `heading` tags to refeer to topics like ‘As Big as It Gets’
image
- Original one
image
- [x] This page should have an `<aside>` tag for Editor's picks section on the righmost side, not a `<div>`. ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/aside_requirement.png)from Pathwright)

- [x] The Elements aligned should have a combination of all three techniques (float, flex and grid). ([Screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_pathwright.png)from Pathwright) and ([Screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_odin.png)from Odin) Looks like you guys forgeted to use the float rule. Kindly implement it 💪
- [x] Your Editor's Pick article have images too small, please make them as the same size of the original one
image
- [x] Your section is not centralized, please fix that 
image
- Original one:
image

#### Good practices REQUIREMENTS (Those changes to make you code like a pro 👽)⤵
- [x] ❌ Your project have linter errors, fix them.
- [x] Rename your classes to make it descriptive (For example: what `.one` means?)
image
- [x] Kindly remember to remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
Take a look at this example:
image
Your HTML will be something like this: 
image
Apply the same logic above in **ALL** those repeated rules listed below:
image

- [x] Your code is throwing some errors and warnings at the [HTML5 Validator](https://validator.w3.org/nu/#textarear), please fix them. 
image

- [x] Please delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */  
- [x] Remove the img folder from css folder.
- [x] Kindly remember to make a descriptive README.md (add a brief description of your project) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59882084/))

## [OPTIONAL CHANGES]
- [ ] Add a favicon to make it more similar to the original page;
- [ ] Improve your names at the REAMDE.md (adding your first and last names)

## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀

  👨🏻‍💻 **TSE: Kevin Alves**

    

  📫 Have any question? Better call Kevin:

  <a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

    
  <sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

File: index.html [hunk 4]
```diff
@@ -0,0 +1,883 @@
...
+          <div class="test">Space Ripples Reveal Big Bang’s Smoking Gun</div>
+        </div>
+      </header>
+    </section>
+
+    <!-- section for the picture -->
+    <div class="picture-container">
+      <div class="main-image">
+        <img src="https://static01.nyt.com/images/2014/03/18/science/space/18cosmos/18cosmos-articleLarge.jpg?quality=75&auto=webp&disable=upscale" alt="" />
+        <!-- <img src="/css/pictures/mainpicture.jpg" alt="" /> -->
```
Comment (kevcap) [inline index.html]: Please remove this commented code.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
# Status: Changes requested :red_circle: 

### Hello @cvilla714 and @abmasadullah :wave: 

### You have done an amazing job implementing almost all of the changes requested by the previous TSE, but we still need to fix these:

- [x] Your linters are still failing, please fix it.

- [x] You still didn't remove the images folder from your css folder, you should put that folder in the root, so your project becomes more organized.

## Kindly make the requested changes and submit for another review, please :pray: 

## I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by elmejdki — APPROVED** [review]
# status: approved :heavy_check_mark: 

now you can merge to master your project is approved

## Happy Coding :tophat:

**Pull request merged**